katharsis-jpa/src/test/java/io/katharsis/jpa/AbstractJpaJerseyTest.java

```diff
@@ -0,0 +1,106 @@
+package io.katharsis.jpa;
+
+import java.util.concurrent.Callable;
+import java.util.concurrent.TimeUnit;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.ws.rs.ApplicationPath;
+import javax.ws.rs.core.Application;
+
+import org.glassfish.jersey.server.ResourceConfig;
+import org.glassfish.jersey.test.JerseyTest;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.katharsis.client.KatharsisClient;
+import io.katharsis.client.ResourceRepositoryStub;
+import io.katharsis.jpa.model.TestEntity;
+import io.katharsis.jpa.query.AbstractJpaTest;
+import io.katharsis.jpa.util.EntityManagerProducer;
+import io.katharsis.jpa.util.SpringTransactionRunner;
+import io.katharsis.jpa.util.TestConfig;
+import io.katharsis.locator.SampleJsonServiceLocator;
+import io.katharsis.queryParams.DefaultQueryParamsParser;
+import io.katharsis.queryParams.QueryParamsBuilder;
+import io.katharsis.rs.KatharsisFeature;
+import io.katharsis.rs.KatharsisProperties;
+
+public class AbstractJpaJerseyTest extends JerseyTest {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add some tests to this class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2187)